iceoryx_utils/source/posix_wrapper/message_queue.cpp

```diff
@@ -400,6 +400,11 @@ cxx::error<IpcChannelError> MessageQueue::createErrorFromErrnum(const std::strin
         std::cerr << "message queue \"" << name << "\" does not exist" << std::endl;
         return cxx::error<IpcChannelError>(IpcChannelError::NO_SUCH_CHANNEL);
     }
+    case ENAMETOOLONG:
+    {
+        std::cerr << "message queue \"" << name << "\" has too long name" << std::endl;
```

Comment:
It sound so german ... maybe 
```cpp
std::cerr << "the message queue name \"" << name << "\" is too long" << std::endl;
```
is better. 

@ithier do you have an even better lyrical euphonious recommendation?

Reply:
My proposal: "The amount of characters in the designation of the message queue entity exceeds the restrictions"

Reply:
@dkroenke my counter proposal (please read it in your head with the voice of Sean Connery):
"Dear Sir or Madam, would you please be so kind to reduce the amount of characters in your splendid message queue name to help me, your humble program, to run your magnificent algorithm. Yours sincerely, RouDi"

Reply:
how about
```
O MessageQueue, MessageQueue, wherefore art thou MessageQueue?
Deny thy request and refuse thy name.
Or if thou wilt not, be but sworn my queue
And I'll no longer be as IPC the same.
```

Reply:
Uuh this is great. I thought also maybe our error messages should sound more sophisticated, articulated, with a glimmer of passion and of course very polite.
`Name too long` does not fulfill then our criteria. We should create an issue for that called lyrical evaluation off end user communication.
@elBoberido @budrus I think this should be the highest prio.